CommandLine/CommandLine.swift

```diff
@@ -569,7 +577,7 @@ public class CommandLine {
       let format = formatOutput != nil ? formatOutput! : defaultFormat
 
       let name = _arguments[0]
-      print(format("Usage: \(name) [options]", .About), terminator: "", toStream: &to)
+      print(format("Usage: \(name) [options]" + (self.suplementalHelp ?? ""), .About), terminator: "", toStream: &to)
```

Comment:
Found out that the original code had support for adding in additional help txt via a formatting block callback, so I did that instead and removed all of this.